src/slices/AccountSlice.ts

```diff
@@ -229,10 +229,10 @@ export const getDonationBalances = createAsyncThunk(
           donationInfo[allDeposits[0][i]] = ethers.utils.formatUnits(allDeposits[1][i], "gwei");
         }
       } catch (e: unknown) {
-        console.error(e);
```

Comment:
@0xLienid Why not keep this error? Also just in case it's not a "user is not donating" error.